traffic_ops/traffic_ops_golang/cdni/shared.go

```diff
@@ -472,12 +491,12 @@ func validateHostExists(host string, tx *sql.Tx) (int, error, error) {
 
 func checkBearerToken(bearerToken string, inf *api.APIInfo) (string, error) {
```

Comment:
It seems this is extracting the `ucdn` claim from the token, but it doesn't appear to verify that the value matches the user's set `ucdn` value in the database - is that necessary?

Reply:
Right but just because it came from us doesn't mean it's up-to-date. If someone's `ucdn` changes since their token was issued, will that cause problems?